README.md

```diff
@@ -1 +1,5 @@
 # canvasCourseDeleteUtility
+
```

Comment:
The only command right now we have is mvn clean install as part of build, If we put that in build.sh then we are not fully utilizing the Jenkins. I will Make a build.sh but i use it after the build is done and to manipulate the jar naming convention.